must-gather/README.md

```diff
@@ -17,7 +17,7 @@ You will get a dump of:
 - All namespaces (and their children objects) that belong to any ocs resources
 - All ocs CRD's definitions
 - All namspaces that contains ceph and noobaa
-- Output of the following ceph commands
+- Output of the following ceph commands in both json and standard format.
```

Comment:
This hunk should go into the patch that changes the collection script to actually do the json collection.